File: C & C++ PRACTICE CODES/space triangle/code8.c
#include <stdio.h>

void main (){

	int row ;
	printf("Enter no. of rows : ");
	scanf("%d",&row);

	for(int i = 0 ; i < row ; i++){

		for(int a = 0 ; a < i ; a++){

			printf("\t");
		}
	
		for(int j = 1; j <= row - i ; j++){

			printf("%d\t",j + i);
		}
		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/7/code8.java
import java.util.Scanner;

class NestedForDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("Enter no. of rows : ");
		int row = s.nextInt();
		int x = 65;

		for(int i = 1; i <= row ; i++){

			for(int j = 0; j < row ; j++){

				System.out.print((char)(x++)+"	");

			}
			System.out.println("\n");

		}

	}


}
